.idea/runConfigurations/Debug_Elasticsearch__node_2_.xml

```diff
@@ -6,6 +6,10 @@
     <option name="HOST" value="localhost" />
     <option name="PORT" value="5008" />
     <option name="AUTO_RESTART" value="true" />
+    <RunnerSettings RunnerId="Debug">
+      <option name="DEBUG_PORT" value="5008" />
+      <option name="LOCAL" value="false" />
+    </RunnerSettings>
```

Comment:
I don't know what makes these! I've made them before on accident. Probably best to revert it. If you can figure out how you made it then we'll see how to include it. Or how to not make it. Or something.